### DISCUSSION TITLE
How to customize the default page for PrivateGPT?

### DISCUSSION THREAD

**Discussion by ruellm** [imartinez/privateGPT#1867] [Q&A]
My question is which file to edit so that the default page when accessing thru localhost:8001 will be customized? i want to edit the color and perhaps re arrange the UI or something.

**Comment by BradHutchings** [comment]
Here you go:

https://github.com/zylon-ai/private-gpt/blob/main/private_gpt/ui/ui.py

It uses Gradio. If you're not familiar with it, you'll quickly become familiar with it. LOL.
